### DISCUSSION TITLE
Error NG05105 found when using toast

### DISCUSSION THREAD

**Discussion by mellunar** [primefaces/community#246] [PrimeNG]
I'm trying the components library for the first time. When I triggered a toast, I got this error:

```
Error: NG05105: Unexpected synthetic listener @toastAnimation.start found. Please make sure that:
  - Either 'BrowserAnimationsModule' or 'NoopAnimationsModule' are imported in your application.
  - There is corresponding configuration for the animation named `@toastAnimation.start` defined in the `animations` field of the '@Component' decorator (see https://angular.io/api/core/Component#animations).
```

If I import any of the mentioned modules, I get another error related to a duplicated load of resources due to `CommonModule` is already imported.

[StackBlitz code](https://stackblitz.com/edit/stackblitz-starters-srjnrp?file=src%2Fpages%2Fhome%2Fhome.page.ts)

**Comment by cetincakiroglu** [comment]
Hi,

I couldn't replicate the issue with the latest PrimeNG, could you please create a new stackblitz example by forking this [demo](https://stackblitz.com/edit/y2cqwm?file=src%2Fapp%2Fdemo%2Ftoast-basic-demo.ts) with fewer pages, no routing etc. because they are irrelevant in this case. It could be because of something missing in your project, otherwise we would receive many issue reports about the error.

**Comment by mellunar** [comment]
I created another environment without routing, and also a clean local installation of Angular CLI 16.2. The same error was logged into the console.

I fixed the issue by adding `importProvidersFrom(BrowserAnimationsModule)` to the `bootstrapApplication` in main.ts. I suggest including this in the documentation for projects that have the standalone syntax.
